src/components/EmployeesList.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+import { getDepartmentById, getEmployeeByDepartmentId } from '../api/api';
+
+import Employee from './Employee';
+import Loader from './Loader';
+
+class EmployeesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      departments: {},
+      employees: [],
+      isLoading: false,
+      error: null,
+    };
+  }
+
+  // async componentDidMount() {
+  //   this.setState({ isLoading: true });
+  //   setTimeout(async () => {
+  //     const departmentId = this.props.match.params.id;
+  //     const [employeesDataError, employeesData] =
+  //       await getEmployeeByDepartmentId(departmentId);
+  //       //console.log(employeesData);
+  //     if (!employeesDataError) {
+  //       this.setState({ employees: employeesData.employeeByDepartmentId });
+  //     } else {
+  //       this.setState({ error: employeesDataError });
+  //     }
+  //     this.setState({ isLoading: false });
+  //   }, 500);
+  // }
+
+  async componentDidMount() {
+    this.setState({ isLoading: true });
+    setTimeout(async () => {
```

Comment:
What is the specific task for setTimeout here?

Reply:
I use the setTimeout to show how spinner would work

Reply:
Got it, but i think it may be removed)))